deployer/aws-lambda/content-origin-response/index.js

```diff
@@ -0,0 +1,13 @@
+exports.handler = async (event) => {
+  /*
+   * Modify the response before it's cached by CloudFront.
```

Comment:
This comment isn't super helpful. 
I think you should replace it with an explanation about the fact that when responses are spawn based on found S3 keys, we can't carry all of their headers, only their things like `cache-control` and that the reason we're setting these here is because it protects us from accidentally uploading an S3 key that might have accidentally missed some important security headers.